digits/dataset/generic/test_views.py

```diff
@@ -231,6 +258,39 @@ def test_edit_notes(self):
             )
         assert status == 200, 'failed with %s' % status
 
+    def test_feature_dims(self):
+        dims = self.get_feature_dims()
+        assert dims == [self.IMAGE_HEIGHT, self.IMAGE_WIDTH, self.CHANNELS]
+
+
+class RandomImageSet(object):
+    """
+    Create a folder of random grayscale images
+    """
+
+    @classmethod
+    def create_imageset(cls, **kwargs):
```

Comment:
Seems simpler to make this a function instead of a method to avoid unnecessary class mixins.